MantidQt/CustomInterfaces/src/Indirect/IndirectDiffractionReduction.cpp

```diff
@@ -270,6 +285,16 @@ void IndirectDiffractionReduction::runGenericReduction(QString instName,
   // Set algorithm properties
   msgDiffReduction->setProperty("Instrument", instName.toStdString());
   msgDiffReduction->setProperty("Mode", mode.toStdString());
+
+  // Check if Cal file is used
+  if (instName == "OSIRIS") {
```

Comment:
Why three nested if statements? Could they not be combined into one? Or collected into a bool with a descriptive name?